docs/section11/Rule11-18.md

```diff
@@ -0,0 +1,42 @@
+# Service_Water_Heating - Rule 11-8
+**Schema Version:** 0.0.37  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-18
+
+**Rule Description:** "Hot-water pumps shall only be enabled when a load exists on the associated hot-water loop"
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** G3.2.3.5  
+
+**Evaluation Context:** each ServiceWaterPiping.  
```

Comment:
G3.2.3.5 applies to space heating HW pumps. Can you please clarify why this is checking SWH systems?

Reply:
it was listed under section 11.  I will move to section 21 & update numbering.